metricbeat/module/kafka/consumer/_meta/docs.asciidoc

```diff
@@ -0,0 +1,20 @@
+This metricset periodically fetches JMX metrics from Kafka Producers JMX.
```

Comment:
We may want to explicitly mention that this only works for consumers implemented in Java, and same thing in the producer metricset.

Btw:
```suggestion
This metricset periodically fetches JMX metrics from Kafka Consumers JMX.
```